### ISSUE TITLE
[CoE Starter Kit - BUG] Default Environment not listed in DLP Impact Tool

### ISSUE THREAD

**Issue by nasilvae** [microsoft/coe-starter-kit#8461]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Team,

can we include "Default" to the drop down environment type selector to the impact tool as this was the original ask? 

https://github.com/microsoft/coe-starter-kit/issues/6555

### Expected Behavior

Default environment should be available in the DLP Impact app

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.31

### What app or flow are you having the issue with?

DLP Impact Analysis

### What method are you using to get inventory and telemetry?

Cloud flows

AB#3553

**Comment by Jenefer-Monroe** [comment]
Default is not excluded I do see it in mine, so perhaps the original issue of too many envts just wasnt correctly addressed. 
Sending to dev for investigation.

**Comment by nasilvae** [comment]
> Default is not excluded I do see it in mine, so perhaps the original issue of too many envts just wasnt correctly addressed. Sending to dev for investigation.

I just replicated it in mines. I searched for the default environment but I don't see it as an option in the combo box to filter it. 

![image](https://github.com/microsoft/coe-starter-kit/assets/95443860/3f2e2f0a-380b-401a-9089-d2b3ae80df7a)

The tenant experiencing the problem operates on a large scale. At present, we're manually embedding the environment into the application to make it functional. Our requirement is simply to include the Default environment SKU in the dropdown menu. This will allow us to select and operate with the default environment without the necessity for manual embedding. The tenant possesses more than 2800 environments, which we believe exceeds the capacity of the environment panel, leading to the current issue.

**Comment by nasilvae** [comment]
We just need the following change :) 
![image](https://github.com/microsoft/coe-starter-kit/assets/95443860/05d23097-5d63-4b14-aa87-ee1b2e76bd69)

**Comment by nasilvae** [comment]
> We just need the following change :) ![image](https://private-user-images.githubusercontent.com/95443860/338675696-05d23097-5d63-4b14-aa87-ee1b2e76bd69.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxMjU3NDcsIm5iZiI6MTcxODEyNTQ0NywicGF0aCI6Ii85NTQ0Mzg2MC8zMzg2NzU2OTYtMDVkMjMwOTctNWQ2My00YjE0LWFhODctZWUxYjJlNzZiZDY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjExVDE3MDQwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE1Y2I3ZWZkMTdiMTQ4YWZhODQzMmViYTM0ZmFkOTU0YzA5YTZhMjllNTVmOWVhZjY1YzcxYjJiYzQzMWM5NDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.N7U8D_KUcHOdK3dRWK-T-odPpyyvCg6gwPz_MMezYxg)

Issue seems to be more complex :( 

![image](https://github.com/microsoft/coe-starter-kit/assets/95443860/1cf2c729-b3c2-4e33-a79c-cfb1dffa91bb)

**Comment by nasilvae** [comment]
I think we need the combo box to reset the query so it reloads the data. We also are seeing teams environments but they are not in the combo box.

**Comment by Jenefer-Monroe** [comment]
oh I see, I was on the wrong page entirely. Yes I dont see teams or default there either
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/5377799d-c3f1-494a-b7db-e8e852653ec9)

To dev for investigation.

**Comment by nasilvae** [comment]
We believe the issue to be with the collections.
![image](https://github.com/microsoft/coe-starter-kit/assets/95443860/9e82d3e4-5aa5-4352-8180-c126ca0d1366)

We are using one collection for all environment types and we are filtering from that collection. We need a collection for each environment type.

**Comment by nasilvae** [comment]
We are modified the original collection to only have the default environment since that is the environment we are currently working on but We believe that if we create a collection for the different environment types we should be good to go. 

col_environments, Filter(Environments, 'Environment Sku' = "Default")

**Comment by Jenefer-Monroe** [comment]
Does it not work to just search for default without the filter at all?
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/67dd3643-7bfe-4de2-8f8e-f03b084f9a91)

**Comment by nasilvae** [comment]
> Does it not work to just search for default without the filter at all? ![image](https://private-user-images.githubusercontent.com/89584709/338711385-67dd3643-7bfe-4de2-8f8e-f03b084f9a91.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxMzMwNTUsIm5iZiI6MTcxODEzMjc1NSwicGF0aCI6Ii84OTU4NDcwOS8zMzg3MTEzODUtNjdkZDM2NDMtN2JmZS00ZGUyLThmOGUtZjAzYjA4NGY5YTkxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjExVDE5MDU1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNlODc1YjZjOTI3ODI0ZTdkYWM4ODhjNjk2ZjdjNDJmODMxYjFkMmIyNjEzYzQyMmVkMjY4NzlkMzg5MWJkODcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eWpoZ037L59JD_ILgnRib56vN9pTzl58pbZE8uPMU9A)

no, We believe that because we have so many environments we go over the limit of the collection.

**Comment by AmarSaiRam** [comment]
Dear User, 

We noticed that this issue has been pending for quite some time, and we have not received any recent updates from your end. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved. 
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue. 
Thank you for your understanding and cooperation. 

Best Regards, 
CoE Starter Kit Team
